src/type/schema.js

```diff
@@ -83,6 +83,15 @@ export function assertSchema(schema: mixed): GraphQLSchema {
  *       directives: specifiedDirectives.concat([ myCustomDirective ]),
  *     })
  *
+ * Note: ObjectTypes which do not have their own queries in the Schema will not end up
+ * in the final Schema. To achieve this, You need to explicitly mention the ObjectType
+ * while creating the Schema. Example:
```

Comment:
@buoyantair Thanks for PR 👍 

> which do not have their own queries in the Schema will not end up in the final Schema

It sounds confising for me. Idea is that schema will include only types that are reachable by traversing root types and only through direct references.
But I'm not sure how to better formulate it.
@Neitsch Can you please help?

I few minor suggestion:
  - Any type not only ObjectTypes so please change `ObjectTypes` => `Types`.
  - would be great to have an example based around interface scenario since it most popular one you can use this code as a basis and remove all unnecessary details but include simplified `Query` definition:
https://github.com/graphql/graphql-js/blob/bdfd1c63e0b45fc48b9c52c2b652680049d76322/src/__tests__/starWarsSchema.js#L299-L302 
  - Can you please move it before note about directive.

Reply:
@IvanGoncharov Updated, do review.